src/test/java/com/deloitte/ddwatch/services/ProjectServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package com.deloitte.ddwatch.services;
+
+
+import com.deloitte.ddwatch.model.Project;
+import com.deloitte.ddwatch.model.ProjectRepo;
+import com.deloitte.ddwatch.repositories.ProjectRepository;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ProjectServiceTest {
+
+    @InjectMocks
+    ProjectService projectService;
+
+    @Mock
+    private ProjectRepository projectRepository;
+
+    @Mock
+    private TagService tagService;
+
+    @Before
+    public void setup() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void createProject() {
+        Project project = Project.builder()
+                .name("test_project")
+                .deliveryLead("")
+                .deliveryLeadEmail("")
+                .technicalLead("")
+                .technicalLeadEmail("")
+                .projectRepos(new HashSet<>())
+                .build();
+
+        when(projectService.create(project))
```

Comment:
Add a test for the other create method which also takes a set of tags as parameter.